lib/elixir/src/elixir_overridable.erl

```diff
@@ -22,10 +22,14 @@ super(Meta, File, Module, Function) ->
   end.
 
 store_pending(Module) ->
-  _ = [{ok, _} = store(Module, X, false) ||
-         {X, {_, _, _, false}} <- maps:to_list(overridable(Module)),
-         not 'Elixir.Module':'defines?'(Module, X)],
-  ok.
+  OverridableFunctionArityPairs = [Pair ||
```

Comment:
I think this can be written as:

```erlang
store_pending(Module) ->
  [begin
    {ok, _} = store(Module, Pair, false),
    Pair
   end || {Pair, {_, _, _, false}} <- maps:to_list(overridable(Module)),
          not 'Elixir.Module':'defines?'(Module, Pair)].
```

This way